pkg/scheduler/scheduler.go

```diff
@@ -319,6 +320,31 @@ func (s *Scheduler) nominate(ctx context.Context, workloads []workload.Info, sna
 	return entries
 }
 
+// resourcesToReserve calculates how much of available resources in cq assignment should be reserved.
+func resourcesToReserve(e *entry, cq *cache.ClusterQueue) cache.FlavorResourceQuantities {
+	if e.assignment.Borrowing || e.assignment.RepresentativeMode() != flavorassigner.Preempt {
```

Comment:
```suggestion
	if e.assignment.RepresentativeMode() != flavorassigner.Preempt {
```

If it's borrowing, but it fits, it gets Fits mode.

Reply:
Because preemption implies reclaiming your nominal quota. Whereas borrowing implies going beyond it. We want to favor the workloads that don't go over nominal.

Reply:
Ack